.env

```
@@ -4,4 +4,10 @@ DB_USER = "root"
 DB_PASSWORD = "Raj@1234"
 DB_NAME = "scibblr"
 JWT_SECRET="scibblr"
-JWT_EXPIRES_IN=2d
\ No newline at end of file
+JWT_EXPIRES_IN=2d
+
+FIREBASE_PROJECT_ID=scribblr-3daf3
+FIREBASE_PRIVATE_KEY="-----BEGIN PRIVATE KEY-----\nMIIEvgIBADANBgkqhkiG9w0BAQEFAASCBKgwggSkAgEAAoIBAQDnf2XMD+qjRphC\nas6krdJLWezsxzkH+hJew3b8hJ2Ny9Yd18S4inh6Cq0j1KSzlhz/XJawsVvO4Ci5\nB1ZeIDu0j2AjoP80/aaY6KNALHJWz8v9sdEzqVTkIoQ2jCyexuX+qws0dYA80DvW\n+ASt8/oKM9M9NL4UgcKL0wv0qd285SxJcW6W+XxtOC1FyxiKU66NhIHG8dpu7A1D\npzAItuleKz9j0mIjPrLvJh0nA9oI0rmRhsEgFlU2Pr+9q1mivdCeNCq2XsClpa4H\nRbEtEI1i1eHuymh3rDiJUqL1b5/Ycms7jQhZErZOP5vvuPFtc5dxa3XlRh3v3XuY\nS7bRwYZFAgMBAAECggEALMWr53qQk8F8gyPrEVh42o/qzaR+ODe6Q8+j/JSdcCYj\nJCRt7I8DW/8im1JuBDmdJBpBHq0+p1UdKyPrJUswoOw12M4zbYk9JkegAzJVbsrD\nli5E2qv+Oh0e2vnjS1YGd2mXQi8TfS9800emiAouqrh5O1YvCQJLoQVRQ6FyvCxs\naOxRkSNqgSZw8d13c0StZEF8wOc2edhI5xh4LzETpB/eCZg0w3l9fVtknRaokkwT\nW/wYd0x57MwAiI7eMZoENhbJpG/WRUeiJ/XX+In4oYtGATMTqTF6zl3zXr3ONari\niQzysmJwhy72H4HzvkKHf3kTEJ88acVTWTLb+KxsRQKBgQDz2QbxIfNwUiOXnuwG\nvBpsMTk6L3WH69RAYn0Gphog+58YjO+6ukQmi6KG2mVXovHT7YLW3en6p8ONWcnW\n8jryM4H7cLov2Dym5mubkWRVf//BCwOLxBXZIBpEbAvX2PqVUGw+nVUyt6eOODIq\nYlg5VvjhxDXFceAtEaIFzcub5wKBgQDzCM9Drwz2QtmQ+uddP2LfVbZTKCi9DSVb\nleb64Lf951BYLAMF29sAQ2xHs72j4skNEIFztoKxQf7qo2Si0FZ6o+oycD/T5S8x\nwLFeP/VPBY6zMGIfAOt2s5dh5Uid8A5Cs7zL7ZrBnyCkLUZdC0NSl0FELKanHq8Z\nwp/Pw93m8wKBgBkkJjAUZQoOnfbS/tjO6hZ/mLvkCsUhznd48BUQaR8/ci7Q0SK6\nc65ZtT8cI1i+xxonodbPeIFMbMAVItdTkDpKjztCax/ltpMqrKuUePjLwq2/mfT+\nGZaSVcL6NNpjV6qvaQmnpRIel1r1dk6iZyA6VFzKoxShYhCjgcYBat2/AoGBANcw\nObTITkXOMy33utwJv9XX4rQGnuultCo/tGD2kHM0WSN49MQ/Qs4c5McoEwYFS9dR\ngg2e0GF6+edv/S9cIPY+JKO8WAeaqgr3fxTgtKffV9a5RGgHPHXB1oxLo9yULUxK\nMSSvoZNG/nPf0qmXy2hZnORsVLMsrAW3GiGwX86ZAoGBAL/Rw5fNAL/Try3p8Tjs\n6p+uK9nE/SznwtGORmV2Sf6HMk6J22U7sxfMFL9Hd3hC36PdLkY26FZAGMVlq4rP\novf4mqA3//0M7LexnaNQNFPPo9St23lbYEm0NK6P2NWn4UYe0OAJ5plcv/riprDK\nooQDR6mTHbEjtagKYRTljm5k\n-----END PRIVATE KEY-----\n",
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The FIREBASE_PRIVATE_KEY key should go before the FIREBASE_PROJECT_ID key

(UnorderedKey)